.env.example

```diff
@@ -0,0 +1 @@
+USER_API_ENDPOINT= #URL of a webhook that contains a JSON formatted array of user data
```

Comment:
Really minor thing - but the comment here still refers to the REST API as a webhook, and in this case I don't think it's quite correct to call it a webhook.

Reply:
Ah, it slipped my mind to change that. I'll merge this and stick it on my next PR